content/react/pt/get-started.md

```diff
@@ -44,34 +44,34 @@ yarn start
 ```
 
 <div class="aside">
-  Nota: Se <code>yarn test</code> emitir um erro, será necessário instalar o pacote <code>watchman</code>, tal como recomendado neste <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">problema</a>
+  Nota: Se o comando <code>yarn test</code> produzir um erro, será necessário instalar o pacote <code>watchman</code>, conforme recomendado nesta <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">thread</a>
 </div>
 
 Sendo estes os seguintes: testes automáticos (Jest), desenvolvimento de componentes (Storybook) e a aplicação em si.
 
 ![3 modalidades](/app-three-modalities.png)
 
 
-Dependendo de qual parte da aplicação que estamos a trabalhar, podemos querer executar um ou mais simultâneamente.
-Visto que neste caso, o foco é a criação de um componente de interface de utilizador simples, iremos cingir-nos somente á execução de Storybook.
+Dependendo de qual parte da aplicação que estamos trabalhando, podemos querer executar um ou mais simultâneamente.
```

Comment:
Why the change to a diferent verb tenses? when the document was created it was done so that a language specificity was implemented. Trying to make as broader as possible